news-frontend/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -52,15 +56,16 @@ const PopupWithForm = (props) => {
         >
           <h2 className="form__title">{props.title}</h2>
           {props.children}
-          <button
-            className="modal__button"
+         <button
+            className={`${ props.isValid ? "modal__button modal__button_active" : "modal__button modal__button_disabled" }`}
             type="submit"
+        
             onClick={props.submit}
```

Comment:
` onClick={props.submit}`

please, delete it

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses

Reply:
thank for the comment, done.